packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -96,6 +96,19 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         : XAxisTypes.StringAxis;
   }
 
+  public componentDidMount(): void {
+    const isChartEmpty = d3Min(this._points, (point: IVerticalBarChartDataPoint) => point.y) === 0;
+    if (isChartEmpty) {
+      d3Select('body')
+        .append('div')
+        .attr('role', 'alert')
+        .attr('id', 'ariaLabel_VerticalBarChart')
+        .style('opacity', 0)
+        .attr('aria-label', 'Graph has no data to display')
+        .attr('tabIndex', 0);
```

Comment:
I have kept the tabIndex for the following reasons:
1. tabIndex=0 will keep the focus on the div in case it is mounted, for the narrator to read the message
2. keeping the tab index will ensure the div is reachable again through the keyboard if the announcement of the chart being empty needs to be narrated again.

Reply:
After attaching the empty aria label component to the chart itself instead of the body, I have removed the tabIndex attribute.